src/components/Menus/Sidebar/InstructionsPanel/InstructionsPanel.jsx

```diff
@@ -147,25 +147,25 @@ const InstructionsPanel = () => {
         instructionsEditable
           ? hasInstructions
             ? [
-                {
-                  className: "btn--secondary",
-                  text: t("instructionsPanel.removeInstructions"),
-                  onClick: () => setShowModal(true),
-                  fill: true,
-                  textAlways: true,
-                  small: true,
-                },
+                <DesignSystemButton
+                  className="btn--secondary"
+                  text={t("instructionsPanel.removeInstructions")}
+                  onClick={() => setShowModal(true)}
+                  fill={true}
+                  textAlways={true}
+                  small={true}
+                />,
               ]
             : [
-                {
-                  className: "btn--primary",
-                  icon: "add",
-                  text: t("instructionsPanel.emptyState.addInstructions"),
-                  onClick: addInstructions,
-                  fill: true,
-                  textAlways: true,
-                  small: true,
-                },
+                <DesignSystemButton
+                  className="btn--primary"
+                  icon="add"
+                  text={t("instructionsPanel.emptyState.addInstructions")}
+                  onClick={addInstructions}
+                  fill={true}
+                  textAlways={true}
+                  small={true}
+                />,
               ]
           : null
```

Comment:
I think it should be empty if there are no buttons rather than null (although I suspect the default on the component is handling this